index.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
+const {execFileSync} = require('child_process');
 const path = require('path');
-const execa = require('execa');
+
+const exec = (cmd, args) => execFileSync(cmd, args || [], {encoding: 'utf8'}).trim();
+
+const execSh = cmd => exec('/bin/sh', ['-c', `"${cmd}"`]);
```

Comment:
This is not the correct way to execute with a shell. Set `{shell: true}` to `execFileSync`.

Reply:
Done. I wasn't aware of the `shell` option, I just copied what `execa` was doing.